scripts/run_tests.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 git clone --depth=1 --branch v3.7.5 https://github.com/jgraph/mxgraph.git
-export PYTEST_ADDOPTS="--driver PhantomJS"
 export MXGRAPHPATH=mxgraph
+
+export LIBGL_ALWAYS_SOFTWARE=1
+export GALLIUM_DRIVER=llvmpipe
+export ENABLE_LLVMPIPE_GL=1
+export QT_QUICK_BACKEND=software
+export QTWEBENGINE_CHROMIUM_FLAGS="--disable-gpu --no-sandbox"
```

Comment:
Why @BeneBr ? When making suggestions like this, make sure to include your reasoning ("I read somewhere", "I know for a fact this is not needed because...", etc).

Reply:
Sorry about that. I used the ```--no-sandbox``` when i had update problems in my S.O. But reading the [docs](https://eden.esss.co/confluence/pages/viewpage.action?pageId=66519393) the ```--no-sandbox``` is not present, so i removed then i all coninued works fine.